test/test_jit.py

```diff
@@ -10481,6 +10481,26 @@ def t(x):
 
         self.checkScript(t, (torch.zeros(3, 2, 3),))
 
+    def test_torch_ignore_conversion_to_none(self):
+        class A(torch.nn.Module):
+            def __init__(self):
+                super(A, self).__init__()
+
+            @torch.jit.ignore
+            def ignored(self, a: int, b: int) -> torch.Tensor:
```

Comment:
Could you give me a bit more context on why it must return Tensor? I think a more pythonic way is return None
```suggestion
            def ignored(self, a: int, b: int) -> None:
```

Reply:
Sorry this was a typo